### ISSUE TITLE
Encrusted Armors might need a rebalance

### ISSUE THREAD

**Issue by Pelirow** [AnonymousProductions/MineFantasy2#22]
There's essentially no point in trying to make encrusted armors when black steel is right around the corner.

Black steel is better in every way and won't cost diamonds, which is much harder to get than obsidian. To make a full black steel armor the main "cost" is in needing to harvest 16 obsidian to make the 32 black steel ingots. Encrusted plate requires 32 diamonds in comparison. I would suggest tweaking the recipe so that 2 encrusted ingots make either 2 or 3 plates as opposed to 1, and the encrusted nuggets make two encrusted chain mesh.

Otherwise, make the encrusted stuff equal  or close to it in armor ratings. Smart players will bypass encrusted armor entirely based on how it's currently set up.

**Comment by AnonymousProductions** [comment]
Hm, could lower the cost by simply giving more shards per diamond, and less dust per shards.

Anyway the reason why there's encrusted (and dragonforge) ingots is so they can use the same recipe, but lessening the cost of the ingot could work, however it would lead to surplus shards for something like a pick

But if you want, change the recipes yourself

**Comment by Pelirow** [comment]
I have no coding skill, if you mean go change it in the code. In terms of recipe, more shards per diamond may work, but like you said, that'd make the encrusted tools "cheaper" although one can argue that making the steel as well balance out the cost. That's why I suggested tweaking only the encrusted chain and plate output, so tools/weapons stay the same.

Dragonforge is hard to figure because I don't have any dragons to fight so I don't know how tough they are compared to 1.6.4 and how many hearts they drop. Assuming they only drop one heart per dragon, then yeah, probably want to tweak the dragonforge ingots a bit too

**Comment by AnonymousProductions** [comment]
ofcourse making a recipe different is annoying for both coding and learning it, sure you only code it once and after learning it it's fine, but keeping the recipes uniform just seems neater.

Ghasts drop glands for now

**Comment by Pelirow** [comment]
I agree keeping recipes uniform is neater and simpler, but you really do need to keep in mind the value of balancing too.

Ask yourself: How much would you use encrusted armors if black steel is just 20 more research points away? Would you spent 24 or 32 diamonds, assuming you even have that many at that point, for a full set of encrusted gear when it won't take much longer to get a clearly better option for less cost?

I mean that literally. Not sure how much you've worked up the armor tiers playing your own mod, but try it yourself and see if you would actually bother to use encrusted armors.

**Comment by AnonymousProductions** [comment]
well it isn't really an official tier, I suppose it can do to be special (like ornate)

**Comment by Pelirow** [comment]
It's a necessary tier- steel doesn't mine obsidian, so an encrusted pick is required. Players need to sink 10 points into encrusted research, unless that was an oversight and steel should mine obsidian. 

Not sure what you can do to make encrusted armors special. Ornate already has very high enchantability as its main perk, right? Perhaps you could make it so that encrusted armors have a chance of causing damage to attackers like the Thorns II/III enchant (level II equivalent for mail, III for plate), but without the durability loss to the armor. Lore could be something about the sharp diamond shards making it unwise for attackers to get close. Was gonna suggest this feature as a possible passive boon for plate armors only, but maybe it could work here instead, exclusive to encrusted stuff.

**Comment by AnonymousProductions** [comment]
I see it as a half tier, still technically being steel
